core/app/models/spree/stock/estimator.rb

```diff
@@ -11,13 +11,13 @@ class OrderRequired < StandardError; end
       #   those marked frontend if truthy
       # @return [Array<Spree::ShippingRate>] the shipping rates sorted by
       #   descending cost, with the least costly marked "selected"
-      def shipping_rates(package, frontend_only = true)
+      def shipping_rates(package, _frontend_only = true)
```

Comment:
do we need to deprecate this argument if it is not used anymore? Maybe also changing method documentation?